Add support for SVG files in get_image_metadata (closes #769)

mre opened the pull request:
This adds support for SVG files to get_image_metadata.
Since the documentation did not mention the supported file formats I did not modify that.
After all it does not change existing behavior.

Sanity check:

* [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

## Code changes
(Delete or ignore this section for documentation changes)

* [X] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [X] Have you created/updated the relevant documentation page(s)?

mre committed:
Add support for SVG files in get_image_metadata

Keats commented on components/templates/src/global_fns/mod.rs:
```diff
@@ -164,14 +166,27 @@ impl TeraFn for GetImageMeta {
+            .map_err(|e| Error::chain(format!("Failed to process image: {}", path.display()), e))?;
+        let h = img.height().ok_or("Undefined height")?;
+        let w = img.width().ok_or("Undefined width")?;
+        Ok((h as u32, w as u32))
```
is that casting a string to u32? If we can't read height/width it's probably better to have an error instead.

Keats replied:
Nevermind, completely forgot you can use `?` on `Option`.

chrisvittal committed:
Reset color after writing message in colorize

Presently when you `^C` in `zola serve` it is painted with the same
color as the previous message. This PR always ensures to reset the color
in colorize, before writing the newline.

Keats committed:
Start 0.10

Keats committed:
Merge pull request #807 from chrisvittal/reset-color-properly

Reset color after writing message in colorize

Keats committed:
Looks like it now requires 1.36

Keats committed:
Add zenburn theme

Closes #813

zaeleus committed:
rendering: Avoid prepending URL prefix to links that start with a scheme (#817)

Links that start with a scheme (e.g., `tel:18008675309`) inadvertently
had a URL prefix prepended. Previously, only `mailto:` was handled, but
given the sheer number of [registered URI schemes][uri-schemes], a loose
pattern matcher is used to detect schemes instead.

External links, as identified by the renderer, are now limited to `http`
and `https` schemes.

Fixes #747 and fixes #816.

[uri-schemes]: https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml

nottxy committed:
Let toc is visable through Page & Section variables in templates (#818)

* Let toc is visable through Page & Section variables in templates

* Removed the current toc variable from page & section

TjeuKayim committed:
Skip anchor checking for URL with prefix in config (#812)

* cargo fmt & clippy

* Skip anchor checking for URL with prefix in config

Keats committed:
Update tera

mre committed:
Update deps

mre committed:
Update cargo.lock
